README.md

```diff
@@ -59,7 +59,11 @@ The integration tests require a different docker-compose stack and a different a
 `npm run start:test`
 
 `npm run int-test(-ui)`
- 
+
+### Building a static page (designers)
+
+To view the existing static pages, and for more instructions about how to edit them, go to `http://localhost:3000/static-pages`.
```

Comment:
I wonder if it's worth warning people (either here or above) that once you try and access `localhost:3000` you get re-routed to auth so you'll need to update your url manually, but maybe it's clear enough 🤷 ?

Reply:
Good shout, added 👍